src/cli.js

```diff
@@ -72,6 +72,8 @@ if (require.main === module) {
     .option('--crash-reporter <value>', 'remote server URL to send crash reports')
     .option('--single-instance', 'allow only a single instance of the application')
     .option('--processEnvs <json-string>', 'a JSON string of key/value pairs to be set as environment variables before any browser windows are opened.', getProcessEnvs)
+    .option('--basic-auth-username <value>', 'basic auth username')
```

Comment:
For clarity: basic ***http(s)*** auth, please

Reply:
Done